### PR TITLE
[IMPROVEMENT] Made variable names compliant

### PR DESCRIPTION
**Pull request by TheClashster**
Please prefix your pull request with one of the following: **[FEATURE]** **[FIX]** **[IMPROVEMENT]**.

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [contributors guide](https://github.com/CCExtractor/sample-platform/blob/master/.github/CONTRIBUTING.md).
- [x] I have checked that another pull request for this purpose does not exist.
- [x] I have considered, and confirmed that this submission will be valuable to others.
- [x] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [x] I give this submission freely, and claim no ownership to its content.

**My familiarity with the project is as follows (check one):**

- [ ] I have never used the project.
- [ ] I have used the project briefly.
- [ ] I have used the project extensively, but have not contributed previously.
- [x] I am an active contributor to the project.

---
No more PyCharm errors.

### CODE REVIEW

File: mod_ci/controllers.py [hunk 1]
```diff
@@ -904,11 +904,11 @@ def blocked_users():
         # Define addUserForm processing
         add_user_form = AddUsersToBlacklist()
         if add_user_form.validate_on_submit():
-            if BlockedUsers.query.filter_by(userID=add_user_form.userID.data).first() is not None:
+            if BlockedUsers.query.filter_by(user_id=add_user_form.user_id.data).first() is not None:
```
Comment (canihavesomecoffee) [inline mod_ci/controllers.py]: This was a careless replace, or you forgot to push your updated forms ;)

File: mod_ci/controllers.py [hunk 2]
```diff
@@ -904,11 +904,11 @@ def blocked_users():
...
                 flash('User already blocked.')
                 return redirect(url_for('.blocked_users'))
 
-            blocked_user = BlockedUsers(add_user_form.userID.data, add_user_form.comment.data)
+            blocked_user = BlockedUsers(add_user_form.user_id.data, add_user_form.comment.data)
```
Comment (canihavesomecoffee) [inline mod_ci/controllers.py]: same here

File: mod_ci/controllers.py [hunk 3]
```diff
@@ -920,7 +920,7 @@ def blocked_users():
                 # Getting all pull requests by blocked user on the repo
                 pulls = repository.pulls.get()
                 for pull in pulls:
-                    if pull['user']['id'] != add_user_form.userID.data:
+                    if pull['user']['id'] != add_user_form.user_id.data:
```
Comment (canihavesomecoffee) [inline mod_ci/controllers.py]: and here ;)

File: mod_ci/controllers.py [hunk 4]
```diff
@@ -948,7 +948,7 @@ def blocked_users():
         # Define removeUserForm processing
         remove_user_form = RemoveUsersFromBlacklist()
         if remove_user_form.validate_on_submit():
-            blocked_user = BlockedUsers.query.filter_by(userID=remove_user_form.userID.data).first()
+            blocked_user = BlockedUsers.query.filter_by(user_id=remove_user_form.user_id.data).first()
```
Comment (canihavesomecoffee) [inline mod_ci/controllers.py]: and here ;)

File: mod_ci/controllers.py (hunk 1 above)
Comment (TheClashster) [reply mod_ci/controllers.py]: Yeah I forgot to stage the form, I had replaced all throughout the repo.

**Pull request merged**